cfme/infrastructure/host.py

```diff
@@ -94,21 +94,25 @@ def __init__(self, **kwargs):
             super(Host.Credential, self).__init__(**kwargs)
             self.ipmi = kwargs.get('ipmi')
 
-    def update(self, updates, validate_credentials=False, from_details=True):
+    def update(self, updates, validate_credentials=False, from_details=True, cancel=False):
         """Updates a host in the UI. Better to use utils.update.update context manager than call
         this directly.
 
         Args:
-           updates (dict): fields that are changing.
+            updates (dict): fields that are changing.
+            validate_credentials (bool): if True, validate host credentials
+            from_details (bool): if True, select 'Edit Selected items' from details view
+                else, select 'Edit Selected items' from hosts view
+            cancel (bool): click cancel button to cancel the edit if True
         """
-        if from_details:
-            view = navigate_to(self, "Edit")
-        else:
+        if not from_details:
```

Comment:
Can you leave the original logic in place here?  I don't see a benefit to switching these, or making this logic negative.

Reply:
I agree. I was trying to keep the logic clearer, but I may have been a bit lazy when I changed the kwarg back to from_details. Updating.